OpenSim/Simulation/Model/Geometry.h

```diff
@@ -232,6 +228,7 @@ class OSIMSIMULATION_API LineGeometry : public Geometry
         upd_end_point() = aPoint2;
     }
 
+protected:
     /** Virtual method to map LineGeometry to SimTK::Array of
     SimTK::DecorativeGeometry.  Appearance, Transforms are handled by base
     Geometry class. */
```

Comment:
@aseth1 Agreed, even though the base class is doing nothing now, it may do in the future.